stan/math/prim/mat/prob/binormal_copula_cdf.hpp

```diff
@@ -0,0 +1,131 @@
+#ifndef STAN_MATH_PRIM_MAT_PROB_BINORMAL_COPULA_CDF_HPP
+#define STAN_MATH_PRIM_MAT_PROB_BINORMAL_COPULA_CDF_HPP
+
+#include <stan/math/prim/scal/meta/is_constant_struct.hpp>
+#include <stan/math/prim/scal/meta/partials_return_type.hpp>
+#include <stan/math/prim/scal/meta/operands_and_partials.hpp>
+#include <stan/math/prim/scal/meta/scalar_seq_view.hpp>
+#include <stan/math/prim/mat/meta/operands_and_partials.hpp>
+#include <stan/math/prim/mat/meta/vector_seq_view.hpp>
+#include <stan/math/prim/scal/err/check_consistent_sizes.hpp>
+#include <stan/math/prim/scal/err/check_not_nan.hpp>
+#include <stan/math/prim/scal/err/check_bounded.hpp>
+#include <stan/math/prim/scal/fun/size_zero.hpp>
+#include <stan/math/prim/scal/fun/value_of_rec.hpp>
+#include <stan/math/prim/scal/err/check_size_match.hpp>
+#include <stan/math/prim/scal/fun/constants.hpp>
+#include <stan/math/prim/scal/fun/value_of.hpp>
+#include <stan/math/prim/scal/meta/size_of.hpp>
+#include <stan/math/prim/scal/meta/max_size.hpp>
+#include <stan/math/prim/scal/meta/contains_nonconstant_struct.hpp>
+#include <boost/random/normal_distribution.hpp>
+#include <stan/math/prim/scal/fun/Phi.hpp>
+#include <stan/math/prim/scal/fun/inv_Phi.hpp>
+#include <stan/math/prim/scal/prob/std_normal_lpdf.hpp>
+#include <stan/math/prim/scal/fun/std_binormal_integral.hpp>
+
+#include <cmath>
+#include <limits>
+#include <algorithm>
+
+namespace stan {
+namespace math {
+
+/**
+ * The CDF of the bivariate normal (binormal) copula for the
+ * specified vector given the specified correlation.
+ *
+ * Reference for the gradients of the function:
+ * Meyer, Christian. "The bivariate normal copula." Communications
+ * in Statistics-Theory and Methods 42, no. 13 (2013): 2402-2422.
+ * https://arxiv.org/pdf/0912.2816.pdf
+ *
+ * @param u 2-vector.
+ * @param rho correlation parameter
+ * for the standard bivariate normal distribution.
+ * @return The probability P(U1 <= u[1], U2 <= u[2] | rho).
+ * @throw std::domain_error if the rho is not between -1 and 1 or nan.
+ * @throw std::domain_error if elements of u aren't between 0 and 1 or nan.
+ * @throw std::invalid_argument u isn't size 2.
+ * @tparam T_u Type of random variable.
+ * @tparam T_rho Type of correlation.
+ */
+
+template <typename T_u, typename T_rho>
+typename return_type<T_u, T_rho>::type binormal_copula_cdf(const T_u& u,
```

Comment:
Should we be calling this `std_binormal_copula_cdf` then?

Reply:
Or I guess people consider the location and scale to be part of the marginal distribution

Reply:
Yeah, that was what I was thinking. I haven't seen the bivariate normal copula as having a nonstandard form.